src/components/mouseposition/MousePositionDirective.js

```diff
@@ -23,6 +25,45 @@ goog.provide('ga_mouseposition_directive');
           control.setProjection(ol.proj.get(projection.value));
           control.setCoordinateFormat(projection.format);
         });
+
+
+        scope.$watch('::ol3d', function(ol3d) {
+          if (ol3d) {
+            var scene = ol3d.getCesiumScene();
+            var ellipsoid = scene.globe.ellipsoid;
+            var handler = new Cesium.ScreenSpaceEventHandler(scene.canvas);
+
+            handler.setInputAction(function(mouvement) {
```

Comment:
This is called alot of times when moving the mouse. Maybe we should throttle this (we have a `gaThrottle` class for this) to 100ms or 200ms.